setup.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env/python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+setup(
```

Comment:
Missing requirements: `click` and `jinja2`. Can't run cmdline tool without these.